src/rating.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+from src.objects import GameRating
+
+def get_rating_arr_for_game(game: np.ndarray, player_ratings):
+    n_players = len(game)
+    rating_arr = []
+    for i in range(n_players):
+        player = game[i]
+        name = player[0]
+        player_rating = get_player_rating_from_name(player_ratings, name)
+        rating_arr.append(GameRating(player, player_rating))
+
+    return rating_arr
+
+def get_player_rating_from_name(ratings, name):
+    n_players = len(ratings)
+
+    for i in range(n_players):
+        if name == ratings[i].name: return ratings[i]
+
+def get_game_rating(game_players: np.ndarray, date):
+    pot_total = calc_pot_total(game_players)
+    n_players = len(game_players)
+    new_ratings = []
+
+    for i in range(n_players):
+        game_player = game_players[i]
+        player_rating = game_player.player_rating
+        player_rating.new_game(date, game_player.net)
+        new_ratings.append(player_rating)
+
+    for i in range(n_players):
```

Comment:
not a massive fan of double nested for loops but not sure how to make any better:)